robyn/__init__.py

```diff
@@ -17,29 +27,34 @@ def start(self, port):
     def get(self, endpoint):
         def inner(handler):
             self.add_route("GET", endpoint, handler)
+
         return inner
-    
+
     def post(self, endpoint):
         def inner(handler):
+            sig = signature(handler)
+            params = len(sig.parameters)
+            if params != 1:
+                print("We need one argument on post.")
+                return
             self.add_route("POST", endpoint, handler)
+
         return inner
 
     def put(self, endpoint):
```

Comment:
@JackThomson2 , I think if we copy your implementation to PUT method, it should also work well imo. What do you think?